Entity/Entities/Bird.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Entities.Interfaces;
+
+namespace Entities.Entities;
+
+public class Bird : IEntity, ICreature, IFlyable
+{
+    public Bird(int id, string name, double maxFlyingHeight)
+    {
+        Id = id;
+        Name = name;
+        MaxFlyingHeight = maxFlyingHeight;
+    }
+
+    public int Id { get; set; }
```

Comment:
you have many common functionality from one class to another, so are you sure that you gonna you interfaces or rather base class?

Reply:
fixed